l10n/en/firefox/privacy-hub.ftl

```diff
@@ -6,24 +6,12 @@
 
 # HTML page title
 firefox-privacy-hub-firefox-privacy-promise = { -brand-name-firefox } Privacy Promise
+firefox-privacy-personal-data-promise = Personal Data Promise
```

Comment:
**Change:** Thank you for following my instructions so carefully... I messed this one up. We can use `firefox-privacy-hub-firefox-privacy-promise` for the h1 and then we don't need to add a new string.